example-workflows/Technique_3_Slack_Incoming_Webhook/blocks.yml

```diff
@@ -1,34 +1,36 @@
-on: [push]
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
 
+name: Post an inline block message
+on:
+  push:
+    branches:
+      - main
 jobs:
-  new_push_job:
+  run:
+    name: Share a travel review to channel
     runs-on: ubuntu-latest
-    name: New push to repo
     steps:
-      - name: Send GitHub trigger payload to Slack Workflow Builder
-        id: slack
+      - name: Write the review
         uses: slackapi/slack-github-action@v2.0.0
         with:
           webhook: ${{ secrets.SLACK_WEBHOOK_URL }}
-          webhook-type: webhook-trigger
+          webhook-type: incoming-webhook
           payload: |
-            "text": "Danny Torrence left a 1 star review for your property."
-            "blocks":
+            text: "Danny Torrence left a 1 star review for your property."
+            blocks:
               - type: "section"
                 text:
                   type: "mrkdwn"
                   text: "Danny Torrence left the following review for your property:"
               - type: "section"
-                block_id: "section567"
-                text: {
+                text:
                   type: "mrkdwn"
-                  text: "<https://example.com|Overlook Hotel> \\n :star: \\n Doors had too many axe holes, guest in room 237 was far too rowdy, whole place felt stuck in the 1920s."
+                  text: "<https://example.com|*Overlook Hotel*> :star: \n Doors had too many axe holes, guest in room 237 was far too rowdy, whole place felt stuck in the 1920s."
                 accessory:
                   type: "image"
                   image_url: "https://is5-ssl.mzstatic.com/image/thumb/Purple3/v4/d3/72/5c/d3725c8f-c642-5d69-1904-aa36e4297885/source/256x256bb.jpg"
                   alt_text: "Haunted hotel image"
               - type: "section"
-                block_id: "section789"
                 fields:
                   - type: "mrkdwn"
-                    text: "*Average Rating*\\n1.0"
+                    text: "*Average Rating*: 1.0"
```

Comment:
but the viewssssss

Reply:
Not so much to see when you're lost in maze! 🦔 ⭐